src/features/search/index.tsx

```
@@ -1,6 +1,54 @@
+            ref={inputRef}
+          />
+          <Button onClick={closePanel} variant="invert" className={s.cancelButton}>
+            <Text type="short-m">{i18n.t('cancel')}</Text>
+          </Button>
+        </div>
+      </Panel>
+      <PanelIcon
+        icon={<Search24 />}
+        className={s.panelIcon}
+        clickHandler={openFullState}
+      ></PanelIcon>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)